.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -461,18 +461,18 @@ jobs:
   cleanup:
     # Deletes internal artifacts
     runs-on: ubuntu-latest
-    needs: [generate-build-definition, build, provenance]
+    needs: [rng, generate-build-definition, build, provenance]
     env:
       BUILD_DEFINITION_NAME: ${{ needs.generate-build-definition.outputs.build-definition-name }}
       SLSA_OUTPUTS_NAME: ${{ needs.build.outputs.slsa-outputs-name }}
     steps:
       - uses: geekyeggo/delete-artifact@54ab544f12cdb7b71613a16a2b5a37a9ade990af # v2.0.0
         with:
-          name: "${{ env.BUILD_DEFINITION_NAME }}-*"
+          name: "${{ env.BUILD_DEFINITION_NAME }}-${{ needs.rng.outputs.value }}"
```

Comment:
I found that this action does not support expanding inputs - so you have to add an env var `RNG: ${{ needs.rng.outputs.value }}` and replace this with `${{ env.RNG }}`

Reply:
Thanks. Done.